docs/learn/learn-staking.md

```diff
@@ -287,49 +287,40 @@ Here is the formula for calculation:
 
     Let x = offenders, n = total no. validators in the active set
 
-    min((3 * (x - (n / 10 + 1))) / n, 1) * 0.07
+    0.05 * min((3 * (x - 1))) / n, 1)
 
 Validators should have a well-architected network infrastructure to ensure the node is running to
-reduce the risk of being slashed or chilled. A high availability setup is desirable, preferably with backup
-nodes that kick in **only once the original node is verifiably offline** (to avoid double-signing
+reduce the risk of being slashed or chilled. A high availability setup is desirable, preferably with 
+backup nodes that kick in **only once the original node is verifiably offline** (to avoid double-signing
 and being slashed for equivocation - see below). A comprehensive guide on validator setup is
 available [here](../maintain/maintain-guides-secure-validator.md).
 
-Validators should have a well-architected network infrastructure to ensure the node is running to
-reduce the risk of being slashed or chilled. A high availability setup is desirable, preferably with backup
-nodes that kick in **only once the original node is verifiably offline** (to avoid double-signing
-and being slashed for equivocation - see below). A comprehensive guide on validator setup is
-available [here](../maintain/maintain-guides-secure-validator.md).
-
-Let us run through a few examples to understand this equation. In all of the examples, assume that
-there are 100 validators in the active set.
-
-> Note that if less than 10% of all validators are offline, no penalty is enacted.
-
-### GRANDPA Equivocation
+Here is an example of unresponsiveness. If there are 100 validators in the active set, 
+and 1 validator is unresponsiveness, the unresponsiveness validator would be slashed at a fraction of 
+0.05 * min((3 * (x - 1))) / n, 1) which comes out to a 5% slash.
```

Comment:
If one validator is unresponsive, then min(0,1) would be 0 and hence no slashing will be applied.

I think a deeper understanding of Slashing mechanisms backed by the code implementation should be used before writing examples for this. Please take some time in drafting the examples.

Reply:
_The parameter of 5% can be adjusted and is 7% in this case, but I am not sure why 7%._

You are not adjusting anything. That is the output of that equation. The maximum anyone can get slashed is 7%. How? if the value is anything greater than 1, then min(value, 1) will return only 1 and subsequently the slashing hits the ceiling of 0.07 (7%)

If you plug in k = n/3 in that equation (assuming you have n = 100 validators), you end up with a value 3 * (33 - (100/ 10 + 1))) / 100) = 3*(33 -11)/100 = 0.66*0.07 which is approximately 5%

Also,

The implementation here is that 10% can be offline and accounts for (n / 10 + 1))), which is subtracted from the total **offending** validators in the active set.

Reply:
Ah, thank you